build.sbt

```diff
@@ -3,10 +3,10 @@ import scala.util.Try
 import scalanative.sbtplugin.ScalaNativePluginInternal._
 import scalanative.io.packageNameFromPath
 
-val sbt13Version          = "0.13.16"
+val sbt13Version          = "0.13.17"
 val sbt13ScalaVersion     = "2.10.7"
-val sbt10Version          = "1.0.4"
-val sbt10ScalaVersion     = "2.12.4"
+val sbt1Version           = "1.2.6"
+val sbt1ScalaVersion      = "2.12.7"
```

Comment:
Lets keep the old `sbt10` suffix to minimize the diff (#1363 has been rebased too many times already).